src/config.default.new

```diff
@@ -0,0 +1,9 @@
+# This is a place for the user to specify their own 'firm' defaults at compile time
```

Comment:
Done.  Don't know how that crept in...

On Tue, Nov 10, 2015 at 10:39 AM Quentin Harley <quentin.harley@gmail.com>
wrote:

> will do.
>
> On Tue, Nov 10, 2015 at 10:15 AM Jim Morris <notifications@github.com>
> wrote:
>
>> In src/config.default.new
>> <https://github.com/Smoothieware/Smoothieware/pull/764#discussion_r44379212>
>> :
>>
>> > @@ -0,0 +1,9 @@
>> > +# This is a place for the user to specify their own 'firm' defaults at compile time
>>
>> what is this? please remove it.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Smoothieware/Smoothieware/pull/764/files#r44379212>.
>>
>